tests/test_handler.py

```diff
@@ -1,27 +1,17 @@
 from collections import defaultdict
 from unittest.mock import MagicMock
-from nio.util.support.block_test_case import NIOBlockTestCase
+from nio.testing.block_test_case import NIOBlockTestCase
 from ..handler import Handler
 
 
 class TestHandler(NIOBlockTestCase):
 
-    def setUp(self):
-        super().setUp()
-        # This will keep a list of signals notified for each output
-        self.last_notified = defaultdict(list)
-
-    def signals_notified(self, signals, output_id='default'):
-        self.last_notified[output_id].extend(signals)
-
     def test_handler_post(self):
-        handler = Handler(endpoint='',
-                          blk=MagicMock())
+        handler = Handler(endpoint='', blk=MagicMock())
```

Comment:
It's a safer/good idea to use the spec attribute of MagicMock so we can catch if we call any methods that don't exist, etc

```python
handler = Handler(endpoint='', blk=MagicMock(spec=WebHandler))
```